---
layout: default
title:  Raphael Lab // Software
container_css_class: container publications
---

# software

### Genetic Heterogeneity: Networks and Pathways

#####CoMEt: Combinations of Exclusive Alterations
CoMEt identifies combinations of exclusive mutations <i>de novo</i> using a statistical score for exclusivity.




<span class="label label-primary">
	<a href="/projects/comet">More information</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/raphael-group/comet/releases">Download (GitHub)</a>
</span>
<span class="label label-success">
	<a href="/projects/comet#reference">Reference</a>
</span>
<span class="label label-warning">
	<a href="https://groups.google.com/forum/#!forum/dendrix">Google Group</a>
</span>

<br/>

#####HotNet2: Network Analysis of Mutation Data
HotNet2 is an algorithm for the discovery of significantly mutated subnetworks in a protein-protein interaction network.

<span class="label label-primary">
	<a href="/projects/hotnet2">More information</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/raphael-group/hotnet2/releases">Download (GitHub)</a>
</span>
<span class="label label-success">
	<a href="/projects/hotnet2#reference">Reference</a>
</span>
<span class="label label-warning">
	<a href="https://groups.google.com/forum/#!forum/hotnet-users">Google Group</a>
</span>

<br/>

#####Multi-Dendrix: (Multiple Pathway De novo Driver Exclusivity)
Multi-Dendrix is an algorithm for the simultaneous discovery of multiple driver pathways using only somatic mutation data from a cohort of samples.

<span class="label label-primary">
	<a href="/projects/multi-dendrix">More information</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/raphael-group/multi-dendrix">Download (GitHub)</a>
</span>
<span class="label label-success">
	<a href="/projects/multi-dendrix#reference">Reference</a>
</span>
<span class="label label-warning">
	<a href="https://groups.google.com/forum/#!forum/dendrix">Google Group</a>
</span>

<br/>

#####Dendrix: De novo Driver Exclusivity
Dendrix is an algorithm for discovery of mutated driver pathways in cancer using only mutation data. 

<span class="label label-primary">
	<a href="/projects/dendrix">More information</a>
</span>
<span class="label label-danger">
	<a href="http://compbio-research.cs.brown.edu/software/Dendrix/Dendrix_v0.3.zip">Download (ZIP)</a>
</span>
<span class="label label-success">
	<a href="/projects/dendrix#reference">Reference</a>
</span>
<span class="label label-info">
	<a href="http://ccmbweb.ccv.brown.edu/dendrix/">Web server</a>
</span>

<br/>

#####HotNet: Finding Altered Subnetworks
HotNet is an algorithm for finding significantly altered subnetworks in a large gene interaction network. 

<span class="label label-primary">
	<a href="/projects/hotnet">More information</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/raphael-group/hotnet/releases">Download (GitHub)</a>
</span>
<span class="label label-success">
	<a href="/projects/hotnet#reference">Reference</a>
</span>
<span class="label label-warning">
	<a href="https://groups.google.com/forum/#!forum/hotnet-users">Google Group</a>
</span>

<br/>

###Structural Variation and Genome Analysis

##### SPRUCE: Somatic Phylogeny Reconstruction using Combinatorial Enumeration #####
Software for inferring the clonal evolution of single-nucleotide and copy-number variants given multi-sample tumor sequencing data.

<span class="label label-primary">
	<a href="/projects/spruce">More information</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/raphael-group/spruce">Download (GitHub)</a>
</span>
<span class="label label-warning">
	<a href="https://groups.google.com/forum/#!forum/spruce">Google Group</a>

<br/>

##### AncesTree #####
Software for inferring the clonal evolution of single-nucleotide variants given multi-sample tumor sequencing data.

<span class="label label-primary">
	<a href="/projects/ancestree">More information</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/raphael-group/AncesTree">Download (GitHub)</a>
</span>
<span class="label label-warning">
	<a href="https://groups.google.com/forum/#!forum/ancestree">Google Group</a>

<br/>

##### NAHR Finder #####
Software for identifying non-allelic homologous recombinations.

<span class="label label-danger">
	<a href="http://compbio-research.cs.brown.edu/software/NAHR/NAHR_finder-0.0.1.tar.gz">Download</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/mmp3/detect-NAHR">Download (GitHub)</a>
</span>

<br/>

#####MultiBreak-SV
Software for structural variation analysis from next-generation paired end data, third-generation
long read data, or data from a combination of sequencing platforms.

<span class="label label-primary">
	<a href="/projects/multibreaksv">More information</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/raphael-group/multibreak-sv">Download (GitHub)</a>
</span>
<span class="label label-success">
	<a href="/projects/multibreaksv#reference">Reference</a>
</span>

<br/>

#####RAIG: Recurrent Aberrations from Interval Graph
RAIG is an algorithm for identifying recurrent and independent copy number aberrations.

<span class="label label-primary">
	<a href="/projects/raig">More information</a>
</span>
<span class="label label-danger">
	<a href="http://compbio-research.cs.brown.edu/software/RAIG/RAIG_1.02.tar.gz">Download (TAR)</a>
</span>
<span class="label label-success">
	<a href="/projects/raig#reference">Reference</a>
</span>

<br/>


#####rec-BTP
rec-BTP is a recursive algorithm for analyzing intra-tumor heterogeneity from high-throughput sequencing data.

<span class="label label-primary">
	<a href="/projects/btp">More information</a>
</span>
<span class="label label-danger">
	<a href="http://compbio-research.cs.brown.edu/software/BTP/rec-btp-1.0.tar.gz">Download (TAR)</a>
</span>
<span class="label label-success">
	<a href="/projects/btp#reference">Reference</a>
</span>

<br/>

#####THetA: Tumor Heterogeneity Analysis
This algorithm estimates tumor purity and clonal/subclonal copy number aberrations directly from high-throughput DNA sequencing data.

<span class="label label-primary">
	<a href="/projects/theta">More information</a>
</span>
<span class="label label-danger">
	<a href="https://github.com/raphael-group/THetA">Download (GitHub)</a>
</span>
<span class="label label-success">
	<a href="/projects/theta#reference">Reference</a>
</span>
<span class="label label-warning">
	<a href="https://groups.google.com/forum/#!forum/theta-users">Google Group</a>
</span>

<br/>

#####PREGO: Paired-End Reconstruction of Genome Organization
This algorithm reconstructs a cancer genome as a rearrangement of segments, or intervals, from the reference genome using paired end sequencing data.

<span class="label label-primary">
	<a href="/projects/prego">More information</a>
</span>
<span class="label label-danger">
	<a href="http://compbio-research.cs.brown.edu/software/PREGO/PREGO_1.2.tar.gz">Download (TAR)</a>
</span>
<span class="label label-success">
	<a href="/projects/prego#reference">Reference</a>
</span>

<br/>

#####GASVPro: Geometric Analysis of Structural Variants
GASVPro is a probabilistic version of our original GASV algorithm.

<span class="label label-primary">
	<a href="/projects/gasv">More information</a>
</span>
<span class="label label-danger">
	<a href="http://code.google.com/p/gasv/">Download (Google Code)</a>
</span>
<span class="label label-success">
	<a href="/projects/gasv#reference">Reference</a>
</span>

<br/>

#####GASV: Geometric Analysis of Structural Variants
Software for analysis of structural variation from paired-end sequencing and/or array-CGH data

<span class="label label-primary">
	<a href="/projects/gasv">More information</a>
</span>
<span class="label label-danger">
	<a href="http://code.google.com/p/gasv/">Download (Google Code)</a>
</span>
<span class="label label-success">
	<a href="/projects/gasv#reference">Reference</a>
</span>

<br/>

#####NBC: Neighborhood Breakpoint Conservation
This software finds recurrent rearrangement breakpoints in DNA copy number data.

<span class="label label-primary">
	<a href="/projects/nbc">More information</a>
</span>
<span class="label label-danger">
	<a href="http://compbio-research.cs.brown.edu/software/NBC/NBCcode.tgz">Download (TAR)</a>
</span>
<span class="label label-success">
	<a href="/projects/nbc#reference">Reference</a>
</span>

<br/>

#####Gremlin: Genome Rearrangement Explorer with Multi-Scale, Linked Interactions
Gremlin is an interactive visualization model for the comparative analysis of structural variation in human and cancer genomes.

<span class="label label-primary">
	<a href="/projects/gremlin">More information</a>
</span>
<span class="label label-danger">
	<a href="/projects/gremlin#download">Download (Coming Soon)</a>
</span>
<span class="label label-success">
	<a href="/projects/gremlin#reference">Reference</a>
</span>

<br/>


###Additional software

#####TADtree
TADtree is an algorithm the identification of hierarchical topological domains in Hi-C data.

<span class="label label-primary">
	<a href="/projects/tadtree">More information</a>
</span>
<span class="label label-danger">
	<a href="/projects/tadtree#download">Download (ZIPs)</a>
</span>

<br/>

#####MAGI: Mutation Annotation & Genome Interpretation
MAGI is a publicly available web application to explore and annotate cancer genomics data.

<span class="label label-primary">
	<a href="http://magi.brown.edu">More information</a>
</span>

<br/>

#####ExaLT: Accurate Genome-Wide Survival Analysis
ExaLT is an algorithm to compute a rigorous approximation to the log-rank p-value that avoids false discoveries compared to standard tools.

<span class="label label-primary">
	<a href="/projects/survival">More information</a>
</span>
<span class="label label-danger">
	<a href="/projects/survival#download">Download (ZIPs)</a>
</span>
<span class="label label-success">
	<a href="/projects/survival#reference">Reference</a>
</span>

<br/>

#####MoDL: Motif Description Length
MoDL finds mutliple motifs in a set of phosphorylated peptides.

<span class="label label-primary">
	<a href="/projects/modl">More information</a>
</span>
<span class="label label-danger">
	<a href="http://compbio-research.cs.brown.edu/software/MoDL/MoDL.tar.gz">Download (TAR)</a>
</span>
<span class="label label-success">
	<a href="/projects/modl#reference">Reference</a>
</span>

<br/>


